_scss/pages/zealery.scss

```diff
@@ -0,0 +1,86 @@
+.page--zealery { //this is to make sure it only shows up on zealery page
+
+  //HERO STYLES
+  h1 {
+    -webkit-text-fill-color: $c-zealery--pink; //overrides color (regardless of order)
+  }
+
+  .hero {
+    background-color: $c-zealery--pink;
+  }
+
+  .hero__img--container {
+    display: none;
+    @media (min-width: $bp-tablet-vert) {
+      display: flex;
+      justify-content: flex-end;
+      width: 105vw;
+    }
+  }
+
+  .hero__img {
+    @media (min-width: $bp-tablet-vert) {
+      position: absolute;
+      &--left {
+        position: absolute;
+        width: 30vw;
+        top: 400px;
+        right: 20%;
+        @media (min-width: $bp-desktop) {
+          top: 450px;
+          right: 20%;
+        }
+      }
+      &--right {
+        position: absolute;
+        width: 40vw;
+      }
+    }
+  }
+
+  //SECTION STYLES
+  .light {
+    .project-section {
+      background-color: $c-zealery--light;
+      h2 {
+        color: $c-zealery--pink;
+      }
+    }
+  }
+
+  .dark {
+    .project-section {
+      background-color: $c-zealery--pink;
+      h2 {
+        color: #fff;
+      }
+      p {
+        color: #fff;
+      }
+    }
+  }
+
+  //WIREFRAMES PROJECT SECTION
+  .wireframes {
+    .section--img2 {
+      padding-top: 75px;
```

Comment:
this is the same as the hifi project section can you consolidate

Reply:
Haven't finished this, I copy+pasted from wireframes->hifi but the positioning of them will be different in the final iteration

Reply:
ok!!